post.md

```diff
@@ -10,13 +10,11 @@ To view the code referenced in this article, checkout the [repo](https://github.
 
 This article and accompanying example application utilize Next.js version 14.x configured with TailwindCSS and TypeScript.
 
-## The Importance of Server-Side Validation
-
-While client-side validation provides immediate feedback and improves user experience, server-side validation is a must for any application. It acts as a second line of defense against invalid or malicious data. Even if you augment your application with client-side validation, server-side validation is necessary to ensure data integrity and security.
-
 ## Overview of Zod Schema for Validation
 
-Zod allows us to define a validation schema for our form data. This schema is a declarative way of specifying the validation rules for each field. For example, to mark a field as required, we can use the `min(1)` method, which specifies that the field must have a minimum length of 1.
+Zod allows us to define a validation schema for our form data. This schema is a declarative way of specifying the validation rules for each field. For example, to mark a field as required, we can use the `min(1)` method, which specifies that the field must have a minimum length of 1. The 2nd argument is optional and can be used to override the default error message.
+
+Zod has many validation methods for different data types and scenarios, for strings you can use the `email()` method to validate an email address or `url()` to validate a URL, if you have custom needs you can always use `regex()` to validate against a regular expression.
```

Comment:
Definitely a well-needed sentence, but it feels a little long. Maybe try braking it it? Something like:

> Zod has many validation methods for different data types and scenarios. You can use the `email()` method to validate an email address, or `url()` to validate a URL. Also, if you have custom needs, you can always use `regex()` to validate against a regular expression.